pkg/webhook/elasticquota/pod_check.go

```diff
@@ -76,3 +78,25 @@ var GetQuotaName = func(pod *corev1.Pod, client client.Client) string {
 	}
 	return extension.DefaultQuotaName
 }
+
+var ListQuotaPods = func(kubeClient client.Client, quota *v1alpha1.ElasticQuota) error {
+	podList := &corev1.PodList{}
+	if err := kubeClient.List(context.TODO(), podList, &client.ListOptions{
+		FieldSelector: fields.OneTermEqualSelector("label.quotaName", quota.Name),
+	}, utilclient.DisableDeepCopy); err != nil {
+		return err
+	}
+	if len(podList.Items) > 0 {
+		return fmt.Errorf("quota has bound pods, isParent is forbidden to modify as true, quotaName: %v", quota.Name)
+	}
+
+	if err := kubeClient.List(context.TODO(), podList, &client.ListOptions{
```

Comment:
I think maybe we should consider opening an internal annotation about namespaces that limit quota usage.

Reply:
will be fixed in another cr.